prov/efa/src/efa_cq.c

```diff
@@ -328,28 +319,9 @@ void efa_cq_poll_ibv_cq(ssize_t cqe_to_process, struct efa_ibv_cq *ibv_cq)
 			break;
 		}
 
-		err = ibv_next_poll(cq->ibv_cq.ibv_cq_ex);
-	}
-
-	if (err && err != ENOENT) {
-		err = err > 0 ? err : -err;
-		prov_errno = ibv_wc_read_vendor_err(cq->ibv_cq.ibv_cq_ex);
-		EFA_WARN(FI_LOG_CQ,
-			 "Unexpected error when polling ibv cq, err: %s (%zd) "
-			 "prov_errno: %s (%d)\n",
-			 fi_strerror(err), err, efa_strerror(prov_errno),
-			 prov_errno);
-		efa_show_help(prov_errno);
-		err_entry = (struct fi_cq_err_entry) {
-			.err = err,
-			.prov_errno = prov_errno,
-			.op_context = NULL,
-		};
-		ofi_cq_write_error(&cq->util_cq, &err_entry);
+		efa_cq_next_poll(ibv_cq);
```

Comment:
ew, you just made an already too deep call stack even deeper.  This can't be the sane answer.